progenitor-impl/src/lib.rs

```diff
@@ -505,41 +505,19 @@ impl Generator {
 
     /// Render text output.
```

Comment:
I think we should just delete generate_text and generatex_text_normalize_comments

Reply:
`generate_text` has been in the `README.md` sample `build.rs`.

Reply:
still! let's get rid of them!